tests/parity_fastapi/benchmark.py

```diff
@@ -0,0 +1,84 @@
+import base64
+import concurrent.futures
+import random
+import time
+
+import requests
+import torch
+from PIL import Image
+import numpy as np
+
+device = "cpu" if torch.cuda.is_available() else "cuda"
+device = "mps" if torch.backends.mps.is_available() else device
+
+rand_mat = np.random.rand(2, 224, 224, 3) * 255
+Image.fromarray(rand_mat[0].astype("uint8")).convert("RGB").save("image1.jpg")
+Image.fromarray(rand_mat[1].astype("uint8")).convert("RGB").save("image2.jpg")
```

Comment:
having this in the run function for accidental importing from this module wont create fake images?